### ISSUE TITLE
Duplicate dat files in ROM/28/ - 41.dat and 41.DAT

### ISSUE THREAD

**Issue by dannygils** [Ashenbubs/AshenbubsHD#3]
ROM/28/ folder appears to contain duplicate files with slightly different naming - 41.DAT and 41.dat. Examined textures in Tex Hammer, both appear the exact same. Same file sizes and md5.

Overwrite confirmation that led me to investigate -
![error](https://user-images.githubusercontent.com/38189995/91663733-bdf42e80-eab8-11ea-9b42-c32998e67e38.png)

md5 - 
![md5](https://user-images.githubusercontent.com/38189995/91663743-cd737780-eab8-11ea-8e87-07108a3430e4.png)

tex hammer of both textures -
![41dat](https://user-images.githubusercontent.com/38189995/91663757-d82e0c80-eab8-11ea-98e9-62bfc8ff7d4e.png)

@Ashenbubs

**Comment by dannygils** [comment]
ID'd another, ROM/32/64.dat & 64.DAT

**Comment by Ashenbubs** [comment]
Removed both these files from GitHub via the website and not the Desktop Tool.

Thanks for spotting these, you're right, they seem to be previously uploaded files that I did before a batch job, however GitHub Desktop doesn't show the duplicates - irritating!  Hopefully it's just these two!

GitHub Desktop View:
![image](https://user-images.githubusercontent.com/61240482/91845431-1100e980-ec51-11ea-9896-e6cf875d3ca5.png)
